.ado/windows-vs-pr.yml

```diff
@@ -276,44 +276,31 @@ jobs:
         timeoutInMinutes: 5 # Set smaller timeout for UTs, since there have been some hangs, and this allows the job to timeout quicker
         condition: and(succeeded(), ne(variables['BuildPlatform'], 'ARM'))
         inputs:
-          testSelector: 'testAssemblies' # Options: testAssemblies, testPlan, testRun
+          testSelector: 'testAssemblies'
           testAssemblyVer2: 'React.Windows.Desktop.UnitTests\React.Windows.Desktop.UnitTests.dll' # Required when testSelector == TestAssemblies
-          #testPlan: # Required when testSelector == TestPlan
-          #testSuite: # Required when testSelector == TestPlan
-          #testConfiguration: # Required when testSelector == TestPlan
-          #tcmTestRun: '$(test.RunId)' # Optional
           searchFolder: '$(Build.SourcesDirectory)\vnext\target\$(BuildPlatform)\$(BuildConfiguration)'
-          #testFiltercriteria: # Optional
-          #runOnlyImpactedTests: False # Optional
-          #runAllTestsAfterXBuilds: '50' # Optional
-          #uiTests: false # Optional
-          #vstestLocationMethod: 'version' # Optional. Options: version, location
-          vsTestVersion: '15.0' # Optional. Options: latest, 15.0, 14.0, toolsInstaller
-          #vstestLocation: # Optional
-          runSettingsFile: $(Build.SourcesDirectory)\vnext\Desktop.UnitTests\tests.runsettings # Optional
-          #overrideTestrunParameters: # Optional
-          #pathtoCustomTestAdapters: # Optional
```

Comment:
Remove all things that are commented out

Reply:
Done.

---
In reply to: [300037626](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r300037626) [](ancestors = 300037626)